doc/source/man.rst

```diff
@@ -327,52 +325,52 @@ A scan can be run by specifying the profile to use and where to write the CSV fi
 
 ``--scan-dirs dir1 dir2``
 
-  The list of directories on remote systems to scan for products. This option is intended to help scope a scan for systems with very large file system under the root directory. You may provide a list of directories or a file where each item is on a separate line.
+  Contains the list of directories on remote systems to scan for products. This option is intended to help scope a scan for systems with a very large file system under the root directory. You can provide multiple values for this option, with each value separated by a space, or provide a path to a file that contains a list of directories, where each directory is on a separate line.
 
 ``--cache``
 
-  This argument can be used if a profile has previously been used for a discovery and nothing new needs to be found during the scan
+  Restricts the scope of the scan to the hosts that were discovered in the previous scan. Use this option to discover software on hosts that were discovered in a previous scan. Do not use this option to scan for new hosts.
 
 ``--vault=vault_file``
 
-  This contains the path of the file that contains the vault password. If this option is used the file should be limited in access on the system.
+  Contains the path of the file that contains the vault password. Because the encrypted Rho data could contain sensitive information, make sure that this vault password file is stored in a location that has limited access.
```

Comment:
`s/Rho/rho/`
I think we are sticking with lowercase `rho` : [see this pr](https://github.com/quipucords/rho/pull/264/files)

Reply:
Thanks for pointing me to that pr. Rho is a tool; rho is a command. So I had an existential crisis regarding referring to Rho or rho for different sentences, and I changed my mind about what to do several times (here's a couple of examples of references to other things that are both tools and commands: http://docs.ansible.com/ansible/latest/ansible-playbook.html and https://git-scm.com/docs/git). If everyone prefers that I stick with rho in all cases to match other Rho internal/developer doc, then I can try to do it in this man page doc. That argument would not win for external/customer doc, though.

Reply:
I leave it to you @jlprevatt. I've leaned to lowercase `rho`, as I dont' really separate the tool and the command, but I just wanna ship it. 😄

Reply:
I agree, you can decide.

Reply:
Actually, I just paged through the entire doc searching for "Rho" and "rho." I think all of the references to "Rho" were accurate in the sense of talking about "this is what the tool does" vs. "this is what the command does." I changed a few references to "the rho command" in paragraphs to include the backwards single quotation marks around "rho" to add highlighting. 

The only thing I didn't touch was the way things are formatted in the Authors section--I'll leave that as it was originally formatted by the creators/contributors.

So I rescind part of my former reply, and just say my existential crisis came back.

Reply:
I appreciate your struggle 😣 Thanks